File: medium/pro-duty-free/duty_free.c
#include <stdio.h>

int main() {
    float item;
    while (1 == scanf("%f", &item)) {
    	if (item * 1.1 > 20) {
    		puts("Back to the store");
    		return 0;
    	}
    }
    puts("On to the terminal");
    return 0;
}

File: medium/pro-duty-free/DutyFree.java
public final class DutyFree {
    public static void main(String[] args) {
        System.out.print(
                new java.util.Scanner(System.in)
                        .useDelimiter(" ")
                        .tokens()
                        .mapToDouble(Double::parseDouble)
                        .map(x -> 1.1 * x)
                        .anyMatch(x -> x > 20)
                        ? "Back to the store" 
                        : "On to the terminal"
        );
    }
}
